superbit_lensing/shear_profiles/make_annular_catalog.py

```diff
@@ -306,7 +306,9 @@ def _compute_metacal_quantities(self):
     def compute_tan_shear_profile(self, outfile, plotfile, overwrite=False, vb=False,
                                   xy_cols=['X_IMAGE', 'Y_IMAGE'],
                                   g_cols=['g1_Rinv', 'g2_Rinv'],
-                                  nfw_center=[5031, 3353]):
+                                  nfw_center=[4940, 3258.5]):
```

Comment:
Yeah, I get that -- this code is still quite messy. `nfw_center` is the centroid used for the shear transformation. It has different values for the coadd data and the `nfw_truth_files`. The `nfw_truth_files` are computed using a single exposure, hence their having a different [X, Y] center from the coadd. 


Here's what I propose:

- Remove the default `nfw_center` in that line in `compute_tan_shear_profile()` altogether. If no image center is provided, then the tangential shear calculation _should_ fail.
- We should rename that `nfw_center` key to something like `image_center` to avoid confusing the coadd image center with the NFW truth file's center. Right now, `nfw_center` is really ambiguous and confusing.
- The image centroid used in the production of the truth NFW file should be embedded in its metadata. That way there is no ambiguity when the [`nfw_info` gets set](https://github.com/superbit-collaboration/superbit-metacal/blob/941326d695659ff9004ffba83e364f4483a2430c/superbit_lensing/shear_profiles/make_annular_catalog.py#L334) in the `compute_tan_shear_profile()` function of  `make_annular_catalog.py`. I'm thinking of something like this: 

```
        if self.nfw_file is not None:

            nfw_truth_tab = Table.read(self.nfw_file, format='fits')
            nfw_truth_xcenter = nfw_truth_tab.meta['x_center']
            nfw_truth_ycenter = nfw_truth_tab.meta['x_center']

            nfw_info = {
                'nfw_file': self.nfw_file,
                'xy_args': ['x_image','y_image'],
                'shear_args': ['nfw_g1','nfw_g2'],
                'image_center': [nfw_truth_xcenter, nfw_truth_ycenter],
            }

```

Reply:
Maybe `image_center` should be called `coadd_center` to be even more clear?